#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("height_20", double_t, 0, "WS09",  0.07, 0.07, 0.30)
gen.add("height_15", double_t, 0, "WS01, WS04",  0.15, 0.07, 0.30)
gen.add("height_10", double_t, 0, "WS03, WS05, WS06, WS10, PP01, CB01, CB02",  0.18, 0.07, 0.30)
gen.add("height_5", double_t, 0, "WS02, WS08, WS12",  0.18, 0.07, 0.30)
gen.add("height_50", double_t, 0, "SH01, SH02",  0.07, 0.07, 0.30)
exit( gen.generate("mir_align_with_workspace", "mir_align_with_workspace", "WorkspaceDistance" ) )
